java/plasma/src/test/java/org/apache/arrow/plasma/PlasmaClientTest.java

```diff
@@ -205,7 +206,32 @@ public void doTest() {
     assert !pLink.contains(id6);
     System.out.println("Plasma java client delete test success.");
     
-    cleanup();
+    // Test calling shuntdown while getting the object.
+    Thread thread = new Thread(() -> {
```

Comment:
for my understanding..is there any reason to call this in a separate thread..can we just call using cleanup?

Reply:
If we call `cleanup` directly, the socket is closed normally. There will be no exception as expected. This exception happens while client is waiting the reply from the server and at this time the server is killed or crashed.

Reply:
Ok, is this test is not susceptible on timing issues then? for e.g. if the read returns before the server cleanup command.

Reply:
Yes, if the read returns before the server cleanup command, this test will fail. However, I used a none-existent object id in the `get` function and the function timeout is 3 seconds to guarantee that this test will pass.